src/quo2/components/list_items/channel/view.cljs

```diff
@@ -0,0 +1,57 @@
+(ns quo2.components.list-items.channel.view
+  (:require [quo2.components.list-items.channel.style :as style]
+            [quo2.theme :as theme]
+            [react-native.core :as rn]
+            [quo2.components.avatars.channel-avatar.view :as channel-avatar]
+            [quo2.components.markdown.text :as quo.text]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.counter.counter.view :as counter]))
+
+(defn- view-internal
+  "Options:
+   - state - (:default/:pressed, default: :default) - show background color when the state is :pressed.
+   - notification - (nil/:notification/:mention/:mute, default: nil):
+     - :notification - Display a grey dot.
+     - :mention - Display a counter.
+     - :mute - Display a mute icon.
+   - locked? - (nil/boolean, default: nil):
+     - When true, display a locked icon.
+     - When false, display an unlocked icon.
+   - mentions-count - (default: nil) - Number of mentions to display in the counter with :mention notification.
+   - customization-color - (default: nil) - Community color.
+   - emoji - (string, default: nil):
+     - Emoji to be displayed on the channel avatar.
+     - If blank, initials of the channel name are displayed.
+   - name - (string, default: nil) - Channel name.
+   - on-press - (function, default: nil) - Function called when the component is pressed.
+   - on-long-press - (function, default: nil) - Function called when the component is long pressed.
+   - theme - Theme value from with-theme HOC"
+  [{:keys [state notification locked? mentions-count customization-color emoji name on-press
+           on-long-press theme]
+    :or   {state :default}}]
+  [rn/pressable
+   {:style               (style/container state customization-color)
+    :accessibility-label :channel-list-item
+    :on-press            on-press
```

Comment:
@ajayesivan can you check out how the quo2 button is handling pressed state and the colors - it uses an overlay. This correctly handles customization color (and community color) 👍

Reply:
Sure. I was going to revisit the pressed state implementation when I saw your comment here https://github.com/status-im/status-mobile/pull/17074#discussion_r1301347478. I will also check the color.

Reply:
@J-Son89 Done

Reply:
nice, curious how did it work considering the values here are opacity 5% etc? all looks good? 🤔